ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/handlers/JsonRpcExecutorHandler.java

```diff
@@ -58,11 +58,13 @@ public static Handler<RoutingContext> handler(
                   } catch (IOException e) {
                     final String method = executor.getRpcMethodName(ctx);
                     LOG.error("{} - Error streaming JSON-RPC response", method, e);
-                    throw new RuntimeException(e);
+                    handleJsonRpcError(ctx, null, RpcErrorType.INTERNAL_ERROR);
```

Comment:
This seems like a good practice, but afaict IOException is going to be thrown by execute in the same thread and the runtime exception would be handled by the outer try/catch.  So I don't think it is strictly necessary.

The change seems ok, but I am curious if Devin gave any justification

Reply:
think you're right on this one, we're not strictly gaining anything except uniformity in the code

Reply:
from Devin - It replaces throwing a new RuntimeException with a call to a method named handleJsonRpcError, which includes the error message from the exception.